### PR TITLE
shell: Label search input

### LINKED ISSUE #13095: Search needs a label
The search currently does not have a label. It should, for accessibility reasons.

```html
<input id="filter-menus" class="form-control" type="search" placeholder="Search">
```

While "search" is referenced several times, the `<input>` still needs a `<label>` or an `aria-label="search"` (which would work well in this case).

For details, see https://dequeuniversity.com/rules/axe/3.4/label

### PR DESCRIPTION
**Pull request by marusak**
Fixes #13095

### CODE REVIEW

File: pkg/shell/index.html [hunk 1]
```diff
@@ -365,7 +365,8 @@ <h4 class="modal-title" translatable="yes">Authentication</h4>
                     </div>
                 </div>
                 <div class="has-feedback search" id="menu-search">
-                    <input id="filter-menus" class="form-control" type="search" placeholder="Search" translate="placeholder">
+                    <input id="filter-menus" class="form-control" type="search" placeholder="Search"
+                        translate="placeholder" aria-label="Search" translate="aria-label">
```
Comment (garrett) [inline pkg/shell/index.html]: You cannot have two of the same attribute in HTML. (The translate attribute is here twice, which is invalid HTML.)

Comment (marusak) [reply pkg/shell/index.html]: hmm, but then we cannot translate the aria-label. I can use then hidden label I guess?

Comment (garrett) [reply pkg/shell/index.html]: Can't it support multiple values? For most other attributes, you just use space separation.

Example: `class="foo bar baz"`

If you use two attributes, then only the last will be set.

Comment (garrett) [reply pkg/shell/index.html]: (Oh, in some browsers, it's the first. In others, it's the last. It's undefined behavior, aside from it being not supported in HTML.)

Comment (marusak) [reply pkg/shell/index.html]: `translate="placeholder aria-label"` picks only `aria-label` for translation. We implement this functionality, so I guess we can fix it? I'll take a look

Comment (marusak) [reply pkg/shell/index.html]: ah no, scratch that, I mistyped and thats why it was not picked up. It works just fine. I pushed it here.

**Pull request merged**